keps/2076-kueuectl/README.md

```diff
@@ -153,6 +154,22 @@ kueuectl create lq|localqueue lqname
 Output:  
 A simple confirmation `localqueue.kueue.x-k8s.io/xxxxx created`
 
+### Create ResourceFlavor
+
+Creates a ResourceFlavor with the given name.
+
+Format:
+
+```
+kueuectl create rf|resourceflavor rfname
+--node-labels key1=value1,key2=value2
+--taints=key1 value:NoSchedule,key2:NoExecute
+--tolerations key1=value:NoSchedule,key2:NoExecute
```

Comment:
```suggestion
--tolerations key1=value:NoSchedule,key2:NoExecute,key3
```

maybe it helps to describe how the parsing works

Reply:
Good point. Thanks.